compiler-plugin-tests/src/test/resources/ballerina_sources/sample_22/service.bal

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2022 WSO2 LLC. (http://www.wso2.com) All Rights Reserved.
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/websubhub;
+import ballerina/http;
+import ballerina/io;
+
+listener http:Listener httpListener = new http:Listener(10012);
+listener websubhub:Listener functionWithArgumentsListener = check new(httpListener, {
+    secureSocket: {
+        key: {
+            path: "tests/resources/ballerinaKeystore.pkcs12",
+            password: "ballerina"
+        }
+    }
+});
+
+service /websubhub on functionWithArgumentsListener {
+    isolated remote function onRegisterTopic(websubhub:TopicRegistration message)
+                                returns websubhub:TopicRegistrationSuccess|websubhub:TopicRegistrationError {
+        if (message.topic == "test") {
+            return websubhub:TOPIC_REGISTRATION_SUCCESS;
+        } else {
+            return websubhub:TOPIC_REGISTRATION_ERROR;
+        }
+    }
+
+    isolated remote function onDeregisterTopic(websubhub:TopicDeregistration message)
+                        returns websubhub:TopicDeregistrationSuccess|websubhub:TopicDeregistrationError {
+        if (message.topic == "test") {
+            return websubhub:TOPIC_DEREGISTRATION_SUCCESS;
+       } else {
+            return websubhub:TOPIC_DEREGISTRATION_ERROR;
+        }
+    }
+
+    isolated remote function onUpdateMessage(websubhub:UpdateMessage msg)
+               returns websubhub:Acknowledgement|websubhub:UpdateMessageError {
+        if (msg.hubTopic == "test") {
+            return websubhub:ACKNOWLEDGEMENT;
+        } else if (!(msg.content is ())) {
+            return websubhub:ACKNOWLEDGEMENT;
+        } else {
+            return websubhub:UPDATE_MESSAGE_ERROR;
+        }
+    }
+    
+    isolated remote function onSubscription(websubhub:Subscription msg)
+                returns websubhub:SubscriptionAccepted|websubhub:SubscriptionPermanentRedirect|websubhub:SubscriptionTemporaryRedirect
```

Comment:
Line exceeds 120 characters

Reply:
check similar places

Reply:
Since these are test resources for compiler-plugin tests do we need to do this ?

Reply:
I think it's always good to follow ballerina [style guide](https://ballerina.io/learn/style-guide/coding-conventions/README/#indentation-and-line-length) when writing ballerina code

Reply:
Will address this in another PR.